docs/source/development/documentation_cookbook.rst

```diff
@@ -0,0 +1,18 @@
+.. _core-development:documentation-cookbook:
+
+***********************
+Documentation cookbook
```

Comment:
As mentioned by @csadorf elsewhere in the PR, I agree that this is not so much for this section but more suited for the wiki.

Reply:
It's just that this is a sphinx cookbook.
It seems weird to put the sphinx cookbook into the wiki.

Reply:
Except that the instructions on how to write documentation for AiiDA, along with all the style guides etcetera _are_ on the wiki. To me it really makes most sense to also add sphinx tips and tricks there.